Intro to Python Notebook #1 .ipynb

```diff
@@ -0,0 +1,1718 @@
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "1.05457e-34\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(h_bar)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Once variables are assigned a value, they can be manipulated however we want: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "6.6260531326e-34\n"
+     ]
+    }
+   ],
+   "source": [
+    "pi = 3.14159\n",
+    "h = 2*pi*h_bar\n",
+    "print(h)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "All variables are a certain type that dictates how it is used. Each type has different properties."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Here are some examples\n",
+    "# By the way, the # symbol is a way to add a comment, these lines will not be run by the \n",
+    "# interpreter, and serve to add context to the code being run \n",
+    "\n",
+    "x=1 #int, only digits\n",
+    "x=1.0 #float, because of the '.', note that re-defining x in this way resets it to a new value\n",
+    "x=1.05457e-34 # also a float because of the '.' and 'e'\n",
+    "x='Duke Devils' #str, defined by quotes around text\n",
+    "x=True #bool"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can check the type of a defined variable with type():"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "bool"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "type(x)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "It is possible to convert between types for example: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "1.0"
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "float(1)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "28"
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "int(\"28\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We have already shown that you can perform multiplication on varialbes. But there are a number of operators that are possible in python. Here are just a few useful ones: \n",
```

Comment:
In this two next blocks I think that using the `` symbol clarifies the description of operators. 

Also, this looks super useful!